scripts/catch/1.9.6/script.sh

```diff
@@ -0,0 +1,31 @@
+MASON_NAME=catch
+MASON_VERSION=1.9.6
+MASON_HEADER_ONLY=true
+
+. ${MASON_DIR}/mason.sh
+
+function mason_load_source {
+    mason_download \
+        https://github.com/philsquared/Catch/archive/v${MASON_VERSION}.tar.gz \
+        c8bf11d32c73c864ebf6b57e2a324a86ea39df82
+
+    mason_extract_tar_gz
+
+    export MASON_BUILD_PATH=${MASON_ROOT}/.build/catch-${MASON_VERSION}
+}
+
+# nothing to build, just copying single include header file
+function mason_compile {
+    mkdir -p ${MASON_PREFIX}/include/
+    cp -r single_include ${MASON_PREFIX}/include
```

Comment:
I think it would be clearer to do `cp -r single_include/catch.hpp ${MASON_PREFIX}/include/`. That way if the layout ever changed, we'd know since this would break. e.g. I like hardcoding that is explicit here for what gets installed. Since the expectation is that this is a single header, we can copy just the single header...